.gitlab-ci.yml

```diff
@@ -15,4 +15,4 @@ libtelio-build-pipeline:
 
   trigger:
     project: $LIBTELIO_BUILD_PROJECT_PATH
-    branch: v1.1.1
+    branch: LLT-5088_moose_5_0_0_upgrade
```

Comment:
Adding a reminder to replace this with the actual tag once the PR is finalized :)